personal_finance.rb

```diff
@@ -0,0 +1,115 @@
+class Transaction
+  attr_accessor :name, :amount, :date, :category, :transactions
+
+  def initialize
+    @name = name
+    @amount = 0
+    @date = Time.now.strftime("%d/%m/%Y %H:%M")
+    @transactions = []
+  end
+
+  finance_on = true
+  while finance_on == true
```

Comment:
You either want to wrap this in a method or place it outside of the class definition. You shouldn't let code hang out inside of a class definition like this unless there's a particularly good reason for it.